lib/builder.js

```diff
@@ -111,7 +112,8 @@ Builder.prototype.build = function (willReadStringTree) {
       name: getDescription(tree),
       broccoliNode: true,
       broccoliId: node.id,
-      broccoliCachedNode: false
+      broccoliCachedNode: false,
+      broccoliPluginName: getPluginName(tree)
```

Comment:
We definitely do not _have_ to, but even if I include the plugin name in the `name` I think I'd still want to have the discreet parts that make up the combination.  For things like summaries by plugin, having them combined into `PluginName: annotation goes here` just means that you have more work to do for both the gathering side (you have to concat them) and the analysis side (you have to split / massage).